src/index.ts

```diff
@@ -37,7 +37,7 @@ import { stopAllServersOnQuit } from './site-server';
 import { loadUserData } from './storage/user-data'; // eslint-disable-next-line import/order
 import { setupUpdates } from './updates';
 
-if ( ! isCLI() ) {
+if ( ! isCLI() && ! process.env.IS_DEV_BUILD ) {
```

Comment:
Do we still need to check `process.env.E2E` in the `enabled` key just below this?

Reply:
We can probably remove it now.
Locally it will fall into the 'development' check, and on CI we have the `IS_DEV_BUILD=true` check. 
Good catch!